beacon-chain/db/deposits.go

```diff
@@ -0,0 +1,53 @@
+package db
+
+import (
+	"fmt"
+
+	"github.com/boltdb/bolt"
+	"github.com/gogo/protobuf/proto"
+	pb "github.com/prysmaticlabs/prysm/proto/beacon/p2p/v1"
+)
+
+// DepositState fetches the deposits state from POW chain from the DB.
+func (db *BeaconDB) DepositState() (*pb.POWDepositState, error) {
+	var powDepositState *pb.POWDepositState
+	err := db.view(func(tx *bolt.Tx) error {
+		powDepInfo := tx.Bucket(depositBucket)
+		enc := powDepInfo.Get(stateLookupKey)
+		if enc == nil {
+			return nil
+		}
+
+		var err error
+		powDepositState, err = createDepositState(enc)
+		return err
```

Comment:
Also, for the documentation can you change it to "deposits state of POW chain from the DB"

Reply:
@IvanTheGreatDev that will not work otherwise `powDepositState` will be initialized as a new variable and will not be assigned to the previously declared var.

Reply:
Ahhhh you're right, my bad

Reply:
Ahhh you're right, my bad

Reply:
thanks for the style advice